packit_service/models.py

```diff
@@ -129,9 +129,22 @@ def get_runs(self) -> List["RunModel"]:
             return trigger_list[0].runs if trigger_list else []
 
     def _get_run_item(self, model):
-        run_ids = [run.id for run in self.get_runs()]
-        with get_sa_session() as session:
-            return session.query(model).filter(RunModel.id.in_(run_ids)).all()
+        runs = self.get_runs()
+        models = []
+
+        if model == CoprBuildModel:
+            models = [run.copr_build for run in runs]
+
+        if model == KojiBuildModel:
+            models = [run.koji_build for run in runs]
+
+        if model == SRPMBuildModel:
+            models = [run.srpm_build for run in runs]
+
+        if model == TFTTestRunModel:
+            models = [run.test_run for run in runs]
+
+        return [model for model in models if model is not None]
```

Comment:
Can we use a different name, please? So we don't collide with the argument...

Reply:
And looking at that, can you please add the type annotation of that function? (Probably my bad, but after some time, it took me a while to get what the `model` argument is about -- maybe, we can rename it to `model_type` or `model_class` to make it obvious.)